src/components/todolistcrud.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable max-classes-per-file */
+import renderTodo from './render.js';
+import TodoStore from './localstorage.js';
+
+class TodoList {
+  constructor(description, isComplete = false) {
+    this.id = TodoStore.getFromStorage().length;
+    this.description = description;
+    this.isComplete = isComplete;
+  }
+}
+
+export default class Crud {
+    static addTodoTask = (event) => {
+      event.target.style.backgroundColor = 'white';
+      const todos = TodoStore.getFromStorage();
+
+      if (event.key === 'Enter') {
+        const description = event.target.value;
+
+        if (document.getElementById('task-desc').value.length === 0) {
+          return;
+        }
+        if (todos.some((task) => task.description === description)) {
+          event.target.style.backgroundColor = '#c76161';
+        } else {
+          todos.push(new TodoList(description));
+          TodoStore.addToStorage(todos);
+          renderTodo();
+          document.getElementById('task-desc').value = '';
+          document.querySelector('.icon').style.color = '#c2b5b5';
+          event.target.style.backgroundColor = 'white';
+        }
+      }
+    }
+
+    static removeTodoTask = (id) => {
+      const todolist = TodoStore.getFromStorage();
+      const filteredlist = todolist.filter((item) => item.id !== id);
+      localStorage.setItem('todos', JSON.stringify(filteredlist));
+    };
```

Comment:
- [ ] Following the project requirements, Deleting a task should update all remaining tasks' Ids, so they represent the current list order and are **unique**. As you see on the screenshot, I deleted `task 2`, I added `task 5` and I noticed that `task 5` and `task 4` have the same id. Kindly work on that.

<img width="958" alt="image" src="https://user-images.githubusercontent.com/77190203/161296534-329de7b0-de1a-4b30-8006-102b6fb71986.png">